flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -291,12 +291,21 @@ object AkkaUtils {
     ConfigFactory.parseString(config)
   }
 
-  def getThreadPoolExecutorConfig: Config = {
+  def getThreadPoolExecutorConfig(threadPriority: Int): Config = {
+    if (threadPriority < Thread.MIN_PRIORITY || threadPriority > Thread.MAX_PRIORITY) {
+      throw new IllegalConfigurationException("The config : " +
+        MetricOptions.QUERY_SERVICE_THREAD_PRIORITY.key() + "'s value must between "
+        + Thread.MIN_PRIORITY + " and " + Thread.MAX_PRIORITY +
+        ", but the value is " + threadPriority)
```

Comment:
This should not refer to the `MetricOptions.QUERY_SERVICE_THREAD_PRIORITY` option. Instead it should say that the `threadPriority` must be between `Thread.MIN_PRIORITY` and `Thread.MAX_PRIORITY`

Reply:
Removed the `MetricOptions.QUERY_SERVICE_THREAD_PRIORITY`.